integration-tests/deployment/ccip/test_helpers.go

```diff
@@ -195,33 +196,53 @@ func NewMemoryEnvironmentWithJobs(t *testing.T, lggr logger.Logger, numChains in
 	return e
 }
 
-func SendRequest(t *testing.T, e deployment.Environment, state CCIPOnChainState, src, dest uint64, testRouter bool) uint64 {
+func CCIPSendRequest(
+	e deployment.Environment,
+	state CCIPOnChainState,
+	src, dest uint64,
+	data []byte,
+	tokensAndAmounts []router.ClientEVMTokenAmount,
+	feeToken common.Address,
+	testRouter bool,
+) (*types.Transaction, uint64, error) {
```

Comment:
This should probably use functional parameters, or accept a parameter object in order to avoid breaking changes in the future when adding things like the strict order flag or a gas limit override.